README.md

```diff
@@ -325,12 +325,50 @@ To install things on the internal MiR PC:
 * connect a monitor and keyboard to the ports that are exposed on one corner of the MiR
 * boot into a live USB linux system
 * `chroot` into the MiR PC
-* download `chrony_2.1.1-1ubuntu0.1_amd64.deb`,
-  `libtomcrypt0_1.17-7ubuntu0.1_amd64.deb`, `libtommath0_0.42.0-1.2_amd64.deb`
-  and `timelimit_1.8-1_amd64.deb` from a PC that has internet and install them
-  in the `chroot` environment onto the MiR PC using `dpkg -i`
+    1. mount MiR partition and bind /dev, /run etc..
+    you can use fdisk -l to figure out which partition to mount.
+    (Here it's _sda3_):
+
+       ```bash
+       sudo mkdir -p /media/mir
+       sudo mount /dev/sda3 /media/mir/
+
+       for dir in /dev /dev/pts /proc /sys /run; do sudo mount --bind $dir /media/mir/@$dir; done
+       ```
+
+    2. `chroot` into the MiR PC:
+       ```sudo chroot /media/mir/@/```
+* create user:
+
+   ```bash
+   adduser newuser
+   usermod -aG sudo newuser
+   ```
+
+* reboot and login into MiR PC
+* Install Chrony
+
+   ```bash
+   sudo apt update
+   sudo apt install chrony
+
+   # if not installable (to fix broken dependencies):
+   sudo apt -f install
+   sudo apt install chrony
+   ```
+
 * set up `/etc/chrony/chrony.conf`
+* make sure all old ntp configs are configured in chrony.
+For this add the following to your chrony.conf (the old ntp.conf part is commented out):
```

Comment:
The Markdown formatting here is broken as well. To fix it, indent the second line by 2 spaces so that it becomes part of the bullet list point above:

```suggestion
  For this add the following to your chrony.conf (the old ntp.conf part is commented out):
```